src/SignalR/server/SignalR/test/Internal/TypedClientBuilderTests.cs

```diff
@@ -92,7 +92,7 @@ public async Task SupportsCancellationToken()
                 send1 =>
                 {
                     Assert.Equal("Method", send1.Method);
-                    Assert.Equal(1, send1.Arguments.Length);
+                    Assert.Single(send1.Arguments);
```

Comment:
Haha, thanks, I've been meaning to fix those warnings 😆

Reply:
Well I was the one who commit that code in the first place